pycbc/waveform/generator.py

```diff
@@ -135,8 +137,13 @@ def dostuff(self):
     def _generate_from_current(self):
         """Generates a waveform from the current parameters.
         """
+        values_hash = hash(frozenset(**self.current_params.values()))
```

Comment:
Why the `frozenset`? I can you just do `hash(self.current_params.values()))`?

Reply:
Actually, thinking about this some more, it would be better to only hash the generator's variable_args. Also, I realized that the hash of a generator doesn't give the correct thing. So:
```
values_hash = frozenset(self.current_params[p] for p in self.variable_args)
```
```